giotto/images/__init__.py

```diff
@@ -0,0 +1,9 @@
+"""The module :mod:`giotto.images` implements preprocessing techniques
+    useful to apply Topological Data Analysis techniques to images.
```

Comment:
Suggestion: "...techniques that can be used to apply Topological Data Analysis to images."